apps/docs/content/tokens/semantic/typography.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Typography
+description: Getting started with Workleap Design Tokens
+---
+
+import tokens from '../../../datas/tokens.json';
+
+# Typography
+
+Typography tokens are composite tokens, meaning they are made up of multiple other tokens. This allows us to create a consistent typographic scale across our products.
+
+## Heading
+
+Headings are used to create a hierarchy of content. They are used to help users scan and understand the content on a page.
+
+<TypographyTable type="heading" data={tokens.semantic} />
```

Comment:
You could still filter it here by sending all the categories you need, like fontSize, fontWeight, fontFamily, lineHeight only

Reply:
i'd avoid logic as much as possible in a .mdx file, since there is no static type checking + no linting